test/unit/manipulation.js

```diff
@@ -3099,3 +3099,66 @@ testIframe(
 		} );
 	}
 );
+
+
+// new test cases
+
+function nodeName( elem, name ) {
+	return elem.nodeName && elem.nodeName.toLowerCase() === name.toLowerCase();
+}
+
+function getAll( context, tag ) {
+
+	// Support: IE <=9 - 11+
+	// Use typeof to avoid zero-argument method invocation on host objects (trac-15151)
+	var ret;
+
+	if ( typeof context.querySelectorAll !== "undefined" ) {
+		ret = context.querySelectorAll( tag || "*" );
+
+	} else {
+		ret = [];
+	}
+
+	if ( tag === undefined || tag && nodeName( context, tag ) ) {
+		return jQuery.merge( [ context ], ret );
+	}
+
+	return ret;
+}
+
+QUnit.test( "should retrieve all elements with a specific tag", function( assert ) {
+    assert.expect( 1 );
+    var context = document.createElement( "div" );
+    context.innerHTML = "<span></span><span></span>";
+
+    var result = getAll( context, "span" );
```

Comment:
This test is calling a locally defined `getAll`; this is not actually testing your changes.

What we need instead is a test confirming the issue that led to this PR is fixed - i.e. #5214.

Reply:
added the test for the issue addressing [#5214](https://github.com/jquery/jquery/issues/5214)